readme.md

```diff
@@ -1,28 +1,36 @@
 [![Discord](https://img.shields.io/discord/725789699527933952)](https://htmx.org/discord)
 
-## Spring Boot integration with HTMX and Thymeleaf
-[HTMX](www.htmx.org) does a great job of making declarative HTML interactive. Thymeleaf is
-good fit since it also focuses on HTML. This project adds processors and other helpers to
-make using both a pleasure.
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/io.github.wimdeblauwe/htmx-spring-boot-thymeleaf/badge.svg)](https://maven-badges.herokuapp.com/maven-central/io.github.wimdeblauwe/htmx-spring-boot-thymeleaf)
+
+# Spring Boot integration with HTMX and Thymeleaf
```

Comment:
I noticed in maven central it is 'Htmx', what is the style guide? Should we do all caps in the readme? I'm fine with either, just want to pick one

Reply:
Good point. Officially, it is all lower case (See https://htmx.org/). I'll adjust it to use that.